spec/src/main/asciidoc/core/scopescontexts.asciidoc

```diff
@@ -362,24 +354,9 @@ The application should not invoke a method of an invalid injected reference. If
 The container must provide an implementation of the `Context` interface for each of the built-in scopes defined in <<builtin_scopes>>.
 These implementations depend on the platform the container is running.
 
-A portable extension may define a custom context object for any or all of the built-in scopes.
-For example, a remoting framework might provide a request context object for the built-in request scope.
-// TODO this refers to Portable Extensions, maybe move to Full? maybe mention Build Compatible Extensions?
-
 The context associated with a built-in normal scope propagates across local, synchronous Java method calls.
 The context does not propagate across remote method invocations or to asynchronous processes.
 
-Portable extensions are encouraged to synchronously fire:
```

Comment:
This should stay in Lite because we want custom contexts to fire these events. You can just reword it as
> Custom contexts are encouraged to....

Reply:
Do we really want BCExtensions to be able to override a built-in context implementation?

Reply:
I mean -- custom contexts, sure. The problem I have is with built-in contexts.

Reply:
> I mean -- custom contexts, sure. The problem I have is with built-in contexts.

Oh, I thought this concerns all custom contexts, my bad!

Reply:
I'll try to find a better place for the custom contexts stuff.